assets/src/infrastructure/ui/inputs/style.scss

```diff
@@ -7,20 +7,50 @@
 .input-select-textarea-focus {
 	border-color: var(--ee-color-primary);
 	box-shadow: 0 0 0 1px var(--ee-color-primary);
+	/* Visible in Windows high-contrast themes */
+	outline-color: transparent;
+	outline-width: 2px;
+	outline-style: solid;
 }
 
-.ee-input-base {
-	&.ee-input-base {
-		border-color: var(--ee-border-color);
-		border-style: solid;
-		border-width: 2px;
-		&:focus {
-			/* Visible in Windows high-contrast themes */
-			outline-color: transparent;
-			outline-width: 2px;
-			outline-style: solid;
-		}
+.base-inputs-styles {
```

Comment:
if this is a new class added by us, then it should be namespaced with `ee-`
I'm guessing it is not since I'm not seeing it applied to a component anywhere

Reply:
good observation, that one is used only to be extended, I also was thinking what is the best approach since in sass docs I saw that they have this prefix for stuff that is extended, but not used standalone:

![image](https://user-images.githubusercontent.com/1477453/80983934-de2b1e00-8e35-11ea-9afc-57b185eb0093.png)

Reply:
lemme know if you want to use that `%` prefix to mark this as being used only for extension

Reply:
link from sass docs where screenie is from: https://sass-lang.com/guide